cmd/kubeadm/app/cmd/phases/init/waitcontrolplane.go

```diff
@@ -93,8 +101,11 @@ func runWaitControlPlanePhase(c workflow.RunData) error {
 	fmt.Printf("[wait-control-plane] Waiting for the kubelet to boot up the control plane as static Pods from directory %q. This can take up to %v\n", data.ManifestDir(), timeout)
 
 	if err := waiter.WaitForKubeletAndFunc(waiter.WaitForAPI); err != nil {
+		runtimeSock := strings.Split(data.Cfg().NodeRegistration.CRISocket, "/")
```

Comment:
i wanted use to long term deprecate this field actually.
/cc @rosti 

i'd prefer if we don't bind more logic to it.

Reply:
umm, were are we gonna store the cri-socket passed via the CLI? Or is it gonna be deprecated fully? 

In that case, Im not sure how to infer which runtime we are using, is kubeadm gonna store that somewhere?

Reply:
I changed this to just print the help commands for both docker and crio+containerd together. That way we dont need to depend on anything else for the moment and maybe we can rework this in the future to use the runtime and print the proper message?